api-spec/v1/transactions-api.yaml

```diff
@@ -1184,6 +1184,10 @@ components:
           description: Circuit unique transaction ID
           type: string
         rrn:
+          description: Transaction rrn
+          type: string
+        validationServiceId:
+          description: Validation service id
```

Comment:
related to bancomatpay flow?

Reply:
No, it's related to MyBank payment flows, this field is used to populate closePayment additionalsData related field